src/app/tests/suites/certification/Test_TC_S_2_5.yaml

```diff
@@ -114,19 +114,19 @@ tests:
       disabled: true
 
     - label: "Step 4a: TH reads from the DUT the SceneTableSize attribute"
-      PICS: S.S.A0006
+      PICS: S.S.A0001
       verification: |
           ./chip-tool scenes read scene-table-size 1 1
 
           [1688019474.695136][1726:1728] CHIP:DMG: }
           [1688019474.695335][1726:1728] CHIP:TOO: Endpoint: 1 Cluster: 0x0000_0005 Attribute 0x0000_0006 DataVersion: 1550229741
-          [1688019474.695397][1726:1728] CHIP:TOO:   SceneTableSize: 0
+          [1688019474.695397][1726:1728] CHIP:TOO:   SceneTableSize: (Default table size: 16)
       disabled: true
 
     - label:
           "Step 4b: TH sends a subscription request action for FabricSceneInfo
           to the DUT."
```

Comment:
Do we need to have PICS flag for the mandatory attributes ?

Reply:
All the other tests still have them, so my understanding is we have them regardless of their mandatory/optional status.